grails-app/i18n/messages_ca.properties

```diff
@@ -103,4 +103,5 @@ upload.instructions.hasList=Atenció\: tornar a enviar una llista existent farà
 
 facet.guid\:null=Tàxons no reconeguts
 
-error.message.reloadListPermission=No teniu permisos per tornar a carregar {0}.
\ No newline at end of file
+error.message.reloadListPermission=No teniu permisos per tornar a carregar {0}.
+australian.iconic.species=Australian iconic species
```

Comment:
This i18n code has been added to all the language files as an english "value". I think the EN value should only be in the first (EN) file and not all the others (i18n falls back to EN when missing value anyway).

Reply:
removed